openedx/core/djangoapps/course_live/views.py

```diff
@@ -207,3 +215,94 @@ def get_provider_data(course_id: str) -> Dict:
                 "available": AVAILABLE_PROVIDERS
             }
         }
+
+
+class CourseLiveIframeView(APIView, LtiCourseLaunchMixin):
+    """
+    A view for retrieving course live iFrame.
+
+    Path: ``api/course_live/iframe/{course_id}/``
+
+    Accepts: [GET]
+
+    ------------------------------------------------------------------------------------
+    GET
+    ------------------------------------------------------------------------------------
+
+    **Returns**
+
+        * 200: OK - Contains a course live zoom iframe.
+        * 401: The requesting user is not authenticated.
+        * 403: The requesting user lacks access to the course.
+        * 404: The requested course does not exist.
+
+    **Response**
+
+        In the case of a 200 response code, the response will be iframe HTML.
+
+    **Example**
+
+        {
+            "iframe": "
+                        <iframe
+                            id='lti-tab-embed'
+                            style='width: 100%; min-height: 800px; border: none'
+                            srcdoc='{srcdoc}'
+                            >
+                        </iframe>
+                        ",
+        }
+
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser
+    )
+    permission_classes = (permissions.IsAuthenticated, IsEnrolled)
+
+    ROLE_MAP = {
+        'student': 'Student',
+        'staff': 'Administrator',
```

Comment:
Let us not remove the instructor role. Map the instructor role to Administrator instead.